src/Types.hs

```diff
@@ -0,0 +1,15 @@
+{-# LANGUAGE OverloadedStrings, QuasiQuotes #-}
+
+module Types where
+
+import           Data.Aeson (FromJSON(parseJSON), Value(..), object, (.=), (.:))
```

Comment:
The word `qualified` fits in the extra space, it's just so package names line up.